### DISCUSSION TITLE
Installing linuxbrew on google cloud gives issue with libunwind.so.8

### DISCUSSION THREAD

**Discussion by alok** [Homebrew/discussions#3780] [Linux]
Every time I try to run a brew command, I get:

`error while loading shared libraries: libunwind.so.8: cannot open shared object file: No such file or directory`

Here is the output of `strace git` for both brewed git (git-bad.txt) and system git (git-good.txt)

[git-bad.txt](https://github.com/Homebrew/discussions/files/9779647/git-bad.txt)
[git-good.txt](https://github.com/Homebrew/discussions/files/9779648/git-good.txt)

**Comment by gromgit** [comment]
What's the output on your Google Cloud instance of:
```
brew config
brew doctor
brew linkage git
ldd `which git`
```
?

**Reply by lmrodriguezr** [reply]
Hello,

I'm experiencing the exact same issue. I've prepared a minimal reproducible notebook for the issue:
https://gist.github.com/lmrodriguezr/23e439bd412d610197ba40b22e70fbdd

Or in Colab:
https://colab.research.google.com/gist/lmrodriguezr/23e439bd412d610197ba40b22e70fbdd/brew_issue.ipynb

I'm illustrating it with `sqlite3` instead of `git` because I've had some unrelated issues with Homebrew's git in that environment, but it can be reproduced with pretty much any binary.

**Reply by lmrodriguezr** [reply]
Sorry, I didn't mention it, but the gist includes the bit requested by @gromgit

Thanks!

**Reply by lmrodriguezr** [reply]
Also, note that Homebrew used to work in that environment without issues, at least up to Oct 2021, as seen in this [older Colab notebook](https://colab.research.google.com/drive/1d4bRsudagAeYqzgmWRicv18c8Y17XEFM).
